src/project/SidebarView.js

```diff
@@ -90,6 +90,7 @@ define(function (require, exports, module) {
      * Show the sidebar.
      */
     function show() {
+	$sidebar.width(176); 
```

Comment:
please fix this indentation - we use two-spaces-per-leve indentation, so either your editor is not set to that, or it's not set to autoconvert "tab" input into spaces =)